pkg/apiserver/api_installer.go

```diff
@@ -104,9 +104,9 @@ func (a *APIInstaller) registerResourceHandlers(path string, storage rest.Storag
 	admit := a.group.Admit
 	context := a.group.Context
 
-	serverGroupVersion := a.group.GroupVersion
+	externalGroupVersion := a.group.GroupVersion
 	if a.group.ServerGroupVersion != nil {
-		serverGroupVersion = *a.group.ServerGroupVersion
+		externalGroupVersion = *a.group.ServerGroupVersion
```

Comment:
In these two places, this is used to get "versioned" options:
https://github.com/kubernetes/kubernetes/pull/18379/files#diff-43c61491ecdb996d7e9eec68fc7f8f36R227
https://github.com/kubernetes/kubernetes/pull/18379/files#diff-43c61491ecdb996d7e9eec68fc7f8f36R265